brands/shop/supermarket.json

```diff
@@ -274,6 +274,15 @@
       "name": "BI-LO",
       "shop": "supermarket"
     }
+  },
+    "shop/supermarket|bi1": {
+    "countryCodes": ["fr"],
+    "nomatch": ["amenity/fuel|bi1"],
+    "tags": {
+      "brand": "bi1",
+      "name": "bi1",
+      "shop": "supermarket"
+    }
```

Comment:
Only having 5 stores probably doesn't make them notable enough to be included. Wikidata will just delete an entry for them at that point. Plus, its a little low for the name index also.